psp/Pv.py

```diff
@@ -574,6 +582,10 @@ def get(self, count=None, timeout=DEFAULT_TIMEOUT, as_string=False, **kw):
         if tmo > 0 and DEBUG != 0:
             logprint("got %s\n" % self.value.__str__())
 
+        if "value" not in self.data:
+            # If the timeout was None, there's a chance that the value hasn't arrived yet.
+            return None
```

Comment:
Conceptual question: what happens if we have self.data and self.value from a previous get? Will this block of code be bypassed and will the function return a stale value?

Reply:
Granted, this isn't a reason not to do this- but if it's true, it should be documented

Reply:
(Also, I think this is how pyca behaves right now if you follow up a successful get with a new timeout=None get, so it's not something in-scope of this PR to modify)

Reply:
Yeah, you are right - if you do `Pv.get(timeout=None)` after `Pv.data` is already populated (via a previous `get()`, or due to monitor callbacks), you'll get the most recently seen value, which might be quite old.  This isn't new behavior to the PR.

I think this is actually the _expected_ outcome with `timeout=None`.  The only element of surprise is that there's now a chance that you might get `None` instead of a real value.